src/Core/src/Platform/Windows/WindowRootView.cs

```diff
@@ -353,5 +337,114 @@ static void OnAppTitleBarTemplateChanged(DependencyObject d, DependencyPropertyC
 		{
 			((WindowRootView)d)._appTitleBar = null;
 		}
+
+		internal static readonly DependencyProperty TitleProperty =
+			DependencyProperty.Register(
+				nameof(WindowTitle),
+				typeof(String),
+				typeof(WindowRootView),
+				new PropertyMetadata(null));
+
+		internal String? WindowTitle
+		{
+			get => (String?)GetValue(TitleProperty);
+			set => SetValue(TitleProperty, value);
+		}
+
+		// Once we switch the TitleBar over to using replaceable IViews we won't need this
+		// but for the sake of first just getting us converted over to the new TitleBar
+		// APIs
+		bool _setTitleBarBackgroundToTransparent = true;
+		internal void SetTitleBarBackgroundToTransparent(bool value)
+		{
+			_setTitleBarBackgroundToTransparent = value;
+			if(value)
+			{
+				if (_appTitleBar is not null && _appTitleBar is Border border)
+					border.Background = null;
```

Comment:
Here is another reference to border? Can this code be refactored to be a method call with the other one so we don't miss it, or is it just coincidence?

Reply:
Moved these to a single method